pkg/processor/errors.go

```diff
@@ -20,6 +20,8 @@ var (
 	// ErrSkipRecord is passed by a processor when it should Ack and skip a Record.
 	// It must be separate from a plain error so that we continue instead of marking
 	// the Pipeline status as degraded.
-	ErrSkipRecord       = cerrors.New("record skipped")
-	ErrInstanceNotFound = cerrors.New("processor instance not found")
+	ErrSkipRecord              = cerrors.New("record skipped")
+	ErrInstanceNotFound        = cerrors.New("processor instance not found")
+	ErrProcessorNotInitialized = cerrors.New("processor not initialized")
```

Comment:
I don't think we need `ErrSkipRecord` and `ErrProcessorNotInitialized`. Well, technically we still need `ErrSkipRecord` because it's used in `FuncWrapper`, but we should rather move to `ProcessorFunc` introduced in the [SDK](https://github.com/ConduitIO/conduit-processor-sdk/pull/17), then we can remove this error.